NRedisTimeSeries/DataTypes/TimeSeriesInformation.cs

```diff
@@ -65,7 +66,9 @@ public class TimeSeriesInformation
         /// </summary>
         public IReadOnlyList<TimeSeriesRule> Rules { get; private set; }
 
-        internal TimeSeriesInformation(long totalSamples, long memoryUsage, TimeStamp firstTimeStamp, TimeStamp lastTimeStamp, long retentionTime, long chunkCount, long chunkSize, IReadOnlyList<TimeSeriesLabel> labels, string sourceKey, IReadOnlyList<TimeSeriesRule> rules)
+        public TsDuplicatePolicy? DuplicatePolicy {  get; private set; }
```

Comment:
why optional and no string?

Reply:
Why string if we have enum for that purpose?